easyfsl/samplers/task_sampler.py

```diff
@@ -80,6 +80,33 @@ def episodic_collate_fn(
                 - their labels,
                 - the dataset class ids of the class sampled in the episode
         """
```

Comment:
(!) If we're going to accept labels of type `Tensor` we need to change the method's signature to 

```python
def episodic_collate_fn(
        self, input_data: List[Tuple[Tensor, Union[Tensor, int]]]
    ) -> Tuple[Tensor, Tensor, Tensor, Tensor, List[int]]:
```

Reply:
Also the expected input and raised errors must be specified in docstring